Blog gatsby cli survey (#23116)
gatsbyjs/gatsby: docs/blog/2020-04-17-gatsby-survey-goes-command-line/index.md

---
title: "Gatsby's First Survey Goes Command Line (And here is what users have to say so far)"
date: 2020-04-16
author: Shannon Soper
excerpt: "When we wanted to survey the community about ways we can make Gatsby better, the command line just felt like the right place to ask. Here's what you told us. (And if you didn't get a chance to respond, we'd still love to hear from you)."
tags:
  - community
  - diversity-and-inclusion
  - gatsby-cloud
---

Some of you have recently seen a link in the CLI, inviting you to take a four question survey about Gatsby:

![Invitation to take Gatsby user survey from the command line](./survey-screenshot.png)

Thank you to all who responded to the survey; myself and other engineers have personally read all the responses and are summarizing them in this blogpost, in an anonymized manner!

> If by chance you have not yet taken the survey, we definitely still want to hear from you! You can submit your own survey response today by upgrading your version of gatsby in a project’s `package.json` file and rerunning dependency and install commands. Or you can install the Gatsby CLI using `npm install -g gatsby-cli` (see the [Quickstart Guide](https://www.gatsbyjs.org/docs/quick-start/) for more details). To trigger the survey, you can run `gatsby feedback` at any time. Otherwise, you’ll be prompted every 3 months, at the most.

Reviewer: ```suggestion
If by chance you have not yet taken the survey, we definitely still want to hear from you! You can submit your own survey response today by upgrading your version of gatsby in a project’s `package.json` file and rerunning dependency and install commands. Or you can install the Gatsby CLI using `npm install -g gatsby-cli` (see the [Quickstart Guide](/docs/quick-start/) for more details). To trigger the survey, you can run `gatsby feedback` at any time. Otherwise, you’ll be prompted every 3 months, at the most.
```

> (If you’d like to opt out, visit https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out).

Reviewer: ```suggestion
(If you’d like to opt-out, visit [our how to opt-out docs](/docs/CLI-feedback/#how-to-opt-out)).
```

2. [Clone this repo](https://github.com/gatsbyjs/error-testing) and find two sites under `error-testing/sites/`. Screen record yourself running one site at a time to test how fast you can resolve the errors in them. Send the screen recording to shannon@gatsbyjs.com!

## How will the data be used?

Anyone in the community, including employees, can read this and all future blog posts reporting results. This way we can publicly compare survey results side-by-side to measure the success of different features; for example, if one of our blog post summaries shows that most people have problems with unclear error messages, and the next post shows they no longer have that problem, all Gatsby community members will know something has changed.

## Thank you and keep the feedback coming!

Thank you again to all who have responded to the survey so far; your responses help guide the focus of the community to answer real-world needs. If you haven’t filled out the survey yet, you can run `gatsby feedback` at any time after upgrading your version of gatsby in a project’s `package.json` file, and then rerunning dependency and install commands.

> Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most. (Again, if you’d like to opt out, visit https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out).

Reviewer: ```suggestion
Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most. (Again, if you’d like to opt-out, visit the [how to opt-out docs](/docs/CLI-feedback/#how-to-opt-out)).
```
